pages/legacyQuestionFile/legacyQuestionFile.js

```diff
@@ -25,16 +26,24 @@ router.get('/:filename', function(req, res, next) {
         if (!result.rows[0].access_allowed) return next(error.make(403, 'Access denied', {locals: res.locals, filename: filename}));
 
         const coursePath = chunks.getRuntimeDirectoryForCourse(course);
-        const chunk = {
-            'type': 'question',
-            'questionId': question.id,
-        };
-        chunks.ensureChunksForCourse(course.id, chunk, (err) => {
+
+        sqldb.call('questions_select_templates', [question.id], (err, result) => {
```

Comment:
Nit: we could probably avoid this call if `!question.template_directory`, right? Probably not worth the effort though, especially since this is implemented with callbacks.

Reply:
Good point, I did this.